media/templates/embed.mustache

```diff
@@ -0,0 +1,281 @@
+{{#>>collection_table}}
```

Comment:
I just followed the collection.mustache example and have it working I think. 3ad69bc